EKF/common.h

```diff
@@ -208,7 +208,7 @@ struct parameters {
 
 	// process noise
 	float gyro_bias_p_noise{1.0e-3f};	// process noise for IMU rate gyro bias prediction (rad/sec**2)
-	float accel_bias_p_noise{3.0e-3f};	// process noise for IMU accelerometer bias prediction (m/sec**3)
+	float accel_bias_p_noise{6.0e-3f};	// process noise for IMU accelerometer bias prediction (m/sec**3)
```

Comment:
Because based on a number of logs there is not enough process noise to enable the bias learning to follow faster temperature induced bias changes during flight. The ideal tuning value is where without protection, it would fall to a value around the target minimum and the protection is there as a backup.